.github/workflows/ci-cd.yml

```
@@ -1,149 +1,89 @@
+  NEXT_PUBLIC_SUPABASE_URL: https://mock-project.supabase.co
+  NEXT_PUBLIC_SUPABASE_ANON_KEY: mock-anon-key
+  NEXT_TELEMETRY_DISABLED: 1
+
 jobs:
-  lint:
+  lint-test-build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)